lib/screen/wallet_connect/wc_connect_page.dart

```diff
@@ -468,69 +486,40 @@ class _WCConnectPageState extends State<WCConnectPage>
-            UIHelper.showGeneratedPersonaDialog(context, onContinue: () {
-              UIHelper.hideInfoDialog(context);
-              final createdPersona = state.persona;
-              if (createdPersona != null) {
-                Navigator.of(context).pushNamed(AppRouter.namePersonaPage,
-                    arguments: createdPersona.uuid);
-              }
-            });
-            break;
-
-          default:
-            break;
-        }
-      },
-      builder: (context, state) {
-        return Column(
+  Widget _createAccountAndConnect() {
+    return Column(
+      children: [
+        const Spacer(),
+        Row(
           children: [
-            // Expanded(
             Expanded(
-              child: Column(
-                children: [
-                  Text("require_full_account".tr(),
-                      //'This service requires a full Autonomy account to connect to the dapp.',
-                      style: theme.textTheme.bodyText1),
-                  const SizedBox(height: 24),
-                  Text(
-                    "generate_full_account".tr(),
-                    //'Would you like to generate a full Autonomy account?',
-                    style: theme.textTheme.headline4,
-                  ),
-                  const SizedBox(height: 24),
-                  Text("newly_account_will".tr(),
-                      //'The newly generated account would also get an address for each of the chains that we support.',
-                      style: theme.textTheme.bodyText1),
-                ],
+              child: AuFilledButton(
+                text: "connect".tr().toUpperCase(),
+                onPress: () {
+                  _createAccount();
+                },
               ),
-            ),
-            // ),
-            Row(
-              children: [
-                Expanded(
-                  child: AuFilledButton(
-                    text: "generate".tr().toUpperCase(),
-                    onPress: () {
-                      context.read<PersonaBloc>().add(CreatePersonaEvent());
-                    },
-                  ),
-                )
-              ],
             )
           ],
-        );
-      },
+        ),
+      ],
     );
   }
+
+  _createAccount() async {
+    UIHelper.showInfoDialog(context, "connecting...".tr(), "");
+    final account = await injector<AccountService>().getDefaultAccount();
+    final defaultName = await account.getAccountDID();
+    var persona =
+        await injector<CloudDatabase>().personaDao.findById(account.uuid);
+    persona!.wallet().updateName(defaultName);
+    final namedPersona = persona.copyWith(name: defaultName);
+    await injector<CloudDatabase>().personaDao.updatePersona(namedPersona);
+    await injector<AuditService>().auditPersonaAction('name', namedPersona);
+    injector<ConfigurationService>().setDoneOnboarding(true);
+    selectedPersona = namedPersona;
+    withDebounce(() => _approveThenNotify(onBoarding: true));
```

Comment:
please move the `withDebounce` to the onPress => withDebounce(() => _createAccount())